prototype-starter/app/views/layout.html

```diff
@@ -1,81 +0,0 @@
-{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the extensions framework. -#}
```

Comment:
In the [notes from 9th Aug](https://docs.google.com/document/d/1hbSiwuUfKpDu91LYRDeUa0yIReYVF5ySsO04kvyOWDA/edit) we talked about keeping `app/views/layout.html`, has there been a change since then?

Reply:
Fair, thank you.  Added.